spec/controllers/external_authors_controller_spec.rb

```diff
@@ -0,0 +1,118 @@
+# frozen_string_literal: true
+require 'spec_helper'
+
+describe ExternalAuthorsController do
+  include LoginMacros
+  include RedirectExpectationHelper
+  let(:user) { FactoryGirl.create(:user) }
+
+  describe "GET #get_external_author_from_invitation" do
+    it "needs a valid invitaiton" do
+      get :get_external_author_from_invitation, invitation_token: "None existent"
+      it_redirects_to_with_error(root_path, "You need an invitation to do that.")
+    end
+   
+    it "needs to have a story assocated with the invitation" do
+      invitation = FactoryGirl.create(:invitation)
+      get :get_external_author_from_invitation, invitation_token: invitation.token
+      it_redirects_to_with_error(signup_path(invitation.token), "There are no stories to claim on this invitation. Did you want to sign up instead?")
+    end
+  end
+
+  describe "GET #complete_claim" do 
+    it "should give the user the works" do
+      external_author = FactoryGirl.create(:external_author)
+      user = FactoryGirl.create(:user)
+      fake_login_known_user(user)
+      invitation = FactoryGirl.create(:invitation, external_author: external_author)
+      get :complete_claim, invitation_token: invitation.token
+      it_redirects_to_with_notice(user_external_authors_path(user), "We have added the stories imported under #{external_author.email} to your account.")
+    end
+  end 
+
+  describe "GET #update" do
+    it "needs to be done by the right user" do
```

Comment:
`it` is for describing expected behavior, often in certain circumstances (`context`). It looks like this is testing that `GET #update` redirects and errors when the user is _not_ right, so it should be more like:
```
context "when user is not right" do
  it "redirects with an error"
```
However, "not right" is not informative. What does a "not right" user look like when we're updating an external author? Code-wise, it looks like `unless (@invitation && @invitation.external_author == @external_author) || @external_author.user == current_user`, but what are we testing here in English?